charts/scalardb-cluster/values.yaml

```diff
@@ -168,7 +168,7 @@ scalardbCluster:
     tag: ""
 
   # -- Optionally specify an array of imagePullSecrets. Secrets must be manually created in the namespace.
-  imagePullSecrets: []
+  imagePullSecrets: [name: reg-docker-secrets]
```

Comment:
Since the ScalarDB Cluster container images are provided from a private container registry, we must set some credentials as a Secret resource to pull those images.

So, I added the default value of the Secret name. This value is based on the existing other charts (e.g., ScalarDL chart).
https://github.com/scalar-labs/helm-charts/blob/scalardl-4.5.1/charts/scalardl/values.yaml#L118